_data/package_index.yml

```
@@ -628,6 +628,12 @@
   tags: blas linear-algebra
   license: BSD-3-Clause
 
+- name: MUMPS
+  github: scivision/mumps
+  description: parallel sparse direct solver, example of injecting CMake in non-CMake source
```

I don't think I would put some build system specific things into the description, what do you think?
```suggestion
  description: parallel sparse direct solver
```